src/phantom/schema.py

```diff
@@ -35,6 +56,9 @@ def __modify_schema__(cls, field_schema: dict) -> None:
         field_schema.update(
             {key: value for key, value in cls.__schema__().items() if value is not None}
         )
+        if cls.__use_docstring__:
+            if ds := desc_from_docstring(cls):
```

Comment:
issue: Unfortunately we cannot use walrus operator yet, phantom-types still supports Python 3.7, see test result and mypy.